examples/pytorch/AxHyperOptimizationPTL/conda.yaml

```diff
@@ -0,0 +1,15 @@
+channels:
+- defaults
+- conda-forge
+- pytorch
+dependencies:
+- python=3.8.2
+- pytorch>=1.6.0
+- torchvision
```

Comment:
Do we need `torchvision`?

Reply:
removed .. cleaned up other packages too.